pkg/proxy/config/config.go

```diff
@@ -63,7 +63,13 @@ type EndpointsConfigHandler interface {
 	// OnEndpointsUpdate gets called when endpoints configuration is changed for a given
 	// service on any of the configuration sources. An example is when a new
 	// service comes up, or when containers come up or down for an existing service.
-	OnEndpointsUpdate(endpoints []api.Endpoints)
+	//
```

Comment:
You should probably state explicitly that it is being passed by reference for efficiency, it won't be obvious to other people reading the code